src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -134,6 +134,10 @@
           type: bool
           short-summary: Don't launch a web browser after establishing port-forwarding.
           long-summary: Add this argument when launching a web browser manually, or for automated testing.
+        - name: --listen-port
+          type: string
```

Comment:
Shouldn't port be a number, not a string?

Reply:
I agree that a port is always said to be a number, though it's used as string here(or else where). If there is any preference for a port argument to be a number, I'm happy to change it. :D

Reply:
The first thing is that you don't need to specify "type: anything" in this file. The CLI literally does not use it for anything.  If the service takes a string and you don't need to do any client-side validation, then this is fine. If you wanted the CLI to validate that the user entered a string, you could use a custom type for that.

Reply:
i see. i'll just remove the type here.